src/Stripe.net/Entities/StripeList.cs

```diff
@@ -1,9 +1,12 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
why is it better to do 3 `using` instead of the specific one that loads just `Generic`?